Marlin/ultralcd.cpp

```diff
@@ -2099,7 +2099,7 @@ void kill_screen(const char* lcd_msg) {
         #endif
       ;
       sprintf_P(UBL_LCD_GCODE, PSTR("G26 C B%i H%i P"), temp, custom_hotend_temp);
-      lcd_enqueue_command_sram("G28");
+      lcd_enqueue_command_sram(PSTR("G28"));
```

Comment:
Since `lcd_enqueue_command_sram` only works with strings that are stored in SRAM, this will not work. Try this instead:
```cpp
char * const g28 = "G28";
lcd_enqueue_command_sram("G28");
```